ISISScientificComputing/autoreduce
# Refactors url declarations from url to path. Moves most runs/ and instrument/ to apps and includes them instead

### Summary of work
- Renames `reduction_variables` to `instrument` to match the urls and what it actually does
- Moves URLs for runs/ and instrument/ into `reduction_viewer.urls` and `instrument.urls`

### How to test your work
Run the web app and check that it starts and all links work.

It's known that the `instrument/MARI/variables/` shows and error but that will be fixed in #1042


### Additional comments
 

Fixes #1100
